openedx/core/djangoapps/user_authn/views/logout.py

```diff
@@ -57,7 +57,11 @@ def dispatch(self, request, *args, **kwargs):
 
         logout(request)
 
-        response = super(LogoutView, self).dispatch(request, *args, **kwargs)
+        # If we are using studio logout directly and there is not OIDC logouts we can just redirect the user
+        if settings.FEATURES.get('DISABLE_STUDIO_SSO_OVER_LMS', False) and not self.oauth_client_ids:
+            response = redirect(self.target)
+        else:
+            response = super(LogoutView, self).dispatch(request, *args, **kwargs)
```

Comment:
Unfortunately, emtiness of the `self.oauth_client_ids` list does not imply that the user is logged out of all IDAs.  IDAs that were converted to use DOT (ecommerce and credentials, on master currently) currently do not get added to that list.

For loggout out the user from those DOT-enabled IDAs, we always unconditionally log out the user from them.  This is a stop-gap solution until we decide on a better/smarter DOT-specific solution.